modules/80_optimization/nash/solve.gms

```diff
@@ -121,13 +122,11 @@ $else.repeatNonOpt
   = smax(regi, p80_repy(regi,"modelstat"));                                    !! also taking into account status 7
 $endif.repeatNonOpt
 
-*** in cm_nash_mode=debug mode, enable solprint for next sol_itr when last
+*** in cm_nash_mode=1 (debug) mode, enable solprint for next sol_itr when last
 *** iteration was non-optimal:
-$ifthen.solprint %cm_nash_mode% == "debug" 
-if (o_modelstat ne 2,   
-    option solprint = on;
+if(cm_nash_mode eq 1 AND o_modelstat ne 2,   
+  option solprint = on;
```

Comment:
Good point. Now, that I look at the code again: why does this solprint statement come so late, after the first sol_itr. When debugging shouldn't it be activated already for the first sol_itr, i.e. in line 14?

Reply:
Ok, since solprint is switched on anyway [in core/loop.gms](https://github.com/dklein-pik/remind/blob/cae76290f089428ddda50f31d27ea94f85c58583/core/loop.gms#L76) (see Oli's comment below), I deleted the above code from [modules/80_optimization/nash/solve.gms](https://github.com/remindmodel/remind/pull/1636/files/cae76290f089428ddda50f31d27ea94f85c58583#diff-afcf543d3c9525ea477e1750ff605c34ad4b854da9f7c01aa06431b321a412a7)